### PR TITLE
fix: update docs

### PR DESCRIPTION
**Pull request by pengx17**

### CODE REVIEW

**Comment by pengx17** [comment]
* **#6094** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6094?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/developing-server.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ MAILER_USER="auth"
 MAILER_PASSWORD="auth"
 MAILER_HOST="localhost"
 MAILER_PORT="1025"
+STRIPE_API_KEY=sk_live_1
```
Comment (forehalo) [inline docs/developing-server.md:64]: add a comment "To debug Payment features"

Comment (pengx17) [reply docs/developing-server.md:64]: This was added to make sure the generated `schema.gql` will include payment related. Not sure if we should ignore schema.gql and related auto-generated files from git?

Comment (forehalo) [reply docs/developing-server.md:64]: that doesn't make sense as well, the schema generation in frontend requires `schema.gql`

Comment (pengx17) [reply docs/developing-server.md:64]: should be related to https://github.com/toeverything/AFFiNE/blob/282b788258ae6f3699cd82a751aeda182fd8cc3d/packages/backend/server/src/plugins/payment/index.ts#L21-L24

Without these two env var, when running `yarn dev` the schema.gql will regenerated and exclude the ones that defined in payment module.

**Review by forehalo — APPROVED** [review]

**Pull request merged**